codequality/spotless/licenseSlashstarStyle.txt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2011-2021 VMware, Inc. or its affiliates, All Rights Reserved.
```

Comment:
the range here should be replaced with $YEAR. this will be interpreted by spotless as follows:

 - single 2021 is present -> all good
 - single year is present, not current year, eg. `2017` -> `2017-2021`
 - range is present ending in 2021 -> all good
 - range is present not ending in 2021, eg. 2017-2019` -> `2017-2021`

Reply:
fixed